infrastructure/Helm/templates/ingress.yaml

```diff
@@ -0,0 +1,25 @@
+apiVersion: networking.k8s.io/v1
+kind: Ingress
+metadata:
+  name: {{ .Values.appname }}-ingress
+  namespace: {{ .Values.namespace}}
+  annotations:
+    kubernetes.io/ingress.class: {{ .Values.ingress.className }}
```

Comment:
**ingress class** annotation is supposed to be deprecated in favour of **spec.ingressClassName** that you have alrady?